src/borg/helpers/parseformat.py

```diff
@@ -624,6 +625,7 @@ def __init__(self, format, repository, manifest, key, *, json=False, iec=False):
             'comment': partial(self.get_meta, 'comment', rs=True),
             'bcomment': partial(self.get_meta, 'comment', rs=False),
             'end': self.get_ts_end,
+            'command_line': partial(self.get_meta, 'cmdline', rs=False),
```

Comment:
maybe just stick with `cmdline` to keep the names in sync?

also, rs=False should be rather `bcmdline`, while `cmdline` would use rs=True.

Reply:
The naming is in sync with the existing `borg info` field. So maybe it's more consistent this way?

The output of `info` might not be binary though.

Reply:
The borg info output is rather a textual description (like "Command line" or "Archive name"), not the short identifier style like used for the placeholders.

So please use cmdline and/or bcmdline here (with the corresponding rs param).

Reply:
> The borg info output is rather a textual description

I meant the `--json` output key which is `command_line` for `info`. So far `info` and `list` JSON keys have been in sync.

Reply:
Looks like `borg info --json` outputs a `command_line` key without calling `remove_surrogates` on it. That's only done for the [none `--json` output.](https://github.com/borgbackup/borg/blob/master/src/borg/archiver.py#L1410)

Renaming the `borg info` key is probably not an option because of backward compatibility? Should I just add `command_line` and `bcommand_line` to both `list` and `info` and make the slight backward-incompatible change to map `remove_surrogates` over `command_line` in `borg info --json`?

Reply:
done.